documentation/configuration-logging.md

```diff
@@ -4,24 +4,27 @@ Logging in our project is done with **Simple Logging Facade for Java (SLF4J)** a
 
 For configuring it, have a look at the `logback*.xml` files in the `[webofneeds/conf](webofneeds/conf)` folder.
 The applications have to be told where to look for this file via a system property that depends on the kind of application.
-* web apps: `-Dlogback.configurationFile=[path-to-your]/logback.xml`
-* spring boot (e.g. bots): `-Dlogging.config=[path-to-your]/logback.xml`
+
+- web apps: `-Dlogback.configurationFile=[path-to-your]/logback.xml`
+- spring boot (e.g. bots): `-Dlogging.config=[path-to-your]/logback.xml`
 
 # Log Levels
 
 The following logging levels are used:
-* **ERROR**: if the application fails to start up, or doesnt't work at all;
-* **WARN**: situations where methods can't execute as they are to be expected (methods called in states where they are not allowed, parametes pointing to not existing atoms, etc.) but the application is still in a operative state; calls, data structures, etc., which go not conform with guidelines, good practices or hard constraints.
-* **INFO**: only in infrequent calls; one time initializations; atom status changes (protocol calls);
-* **DEBUG**: everything else.
+
+- **ERROR**: if the application fails to start up, or doesnt't work at all;
+- **WARN**: situations where methods can't execute as they are to be expected (methods called in states where they are not allowed, parametes pointing to not existing atoms, etc.) but the application is still in a operative state; calls, data structures, etc., which go not conform with guidelines, good practices or hard constraints.
+- **INFO**: only in infrequent calls; one time initializations; atom status changes (protocol calls);
+- **DEBUG**: everything else.
```

Comment:
**TRACE** is also available, descriptions seem rather odd(especially error) idk if we want to have any of this in our documentation even, these describe very common loglevels only and depend on the specific logger method call chosen by the developer and not the severity of a problem. i think a reference to log4j or loglevel documentation is more than sufficient

Reply:
let's skip that.

Reply:
replaced with links.